### PR TITLE
Added python quiz question

### PR DESCRIPTION
**Pull request by SharoonRafeek**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+   {
+    Question: 
+      "In Python, what is the output of the following code print(list(range(3, 6)))?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:4]:
```suggestion
      "In Python, what is the output of the following code: print(list(range(3, 6)))?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by SharoonRafeek** [comment]
I have made all the necessary changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**